tests/README.md

```diff
@@ -36,3 +36,9 @@ make test-integration
 ```
 
 Note that this also runs the runtime tests as well.
+
+To run all the tests that include external dependencies use:
+
+```bash
+make test-integration-full
+```
```

Comment:
This is going to conflict with https://github.com/fermyon/spin/pull/2268 which includes a bit more detail

Reply:
I will remove this.